### ISSUE TITLE
[REQ]Change % value to represent based on current threat holder

### ISSUE THREAD

**Issue by Stryker159** [EsreverWoW/ClassicThreatMeter#8]
Currently when reaching the top of threat, it lists that player as the 100% value even if they have not pulled threat at that point. As a class at range can go to 130% threat before pulling aggro can this value be changed to represent your % of threat compared to the current threat holder.

I remember this being part of the threat mods back in vanilla WoW back in the day

**Comment by EsreverWoW** [comment]
You are correct in that ranged has 130% and melee has 110% for an aggro pull, but my aim was to make default normalized values so that 100% means pull, however this only really works when values are correct in calculations and you don't have things such as #9 going on where resisted spells are still counted when they shouldn't be. I will be making normalized values optional and allow for the more traditional 130% bar that many players are accustomed to in a future release.

As we obtain further research into correct values, the normalization will be more useful, but this early on it will likely cause more confusion than anything.
